pandas/io/tests/parser/test_textreader.py

```diff
@@ -392,6 +393,16 @@ def test_empty_field_eof(self):
                           names=list('abcd'), engine='c')
             assert_frame_equal(df, c)
 
+    def test_empty_csv_input(self):
+        # GH14867
+        df1 = read_csv(StringIO(), chunksize=20, header=None,
```

Comment:
I would just write a temporary blank file here instead of adding an external file (or even better, just use a StringIO)